tests/common/plan_stubs/test_set_and_do_other_plan.py

```diff
@@ -0,0 +1,145 @@
+from collections import defaultdict
+from unittest.mock import AsyncMock
+
+import pytest
+from bluesky.plans import count
+from bluesky.protocols import Location, Reading
+from bluesky.run_engine import RunEngine
+from dodal.devices.motors import XYZStage
+from ophyd_async.core import init_devices
+from ophyd_async.epics.core import epics_signal_rw
+from ophyd_async.testing import assert_emitted
+
+from sm_bluesky.common.plan_stubs.set_and_do_other_plan import (
+    set_and_wait_within_tolerance,
+)
+
+
+@pytest.fixture
+async def sim_rw_signal():
+    async with init_devices(mock=True):
+        sim_rw_signal = epics_signal_rw(float, read_pv="BL007", write_pv="write")
+
+    return sim_rw_signal
```

Comment:
`from ophyd_async.core import SignalRW`

```suggestion
@pytest.fixture
async def sim_rw_signal() -> SignalRW[float]:
    async with init_devices(mock=True):
        sim_rw_signal = epics_signal_rw(float, read_pv="BL007", write_pv="write")
    return sim_rw_signal
```